### ISSUE TITLE
stopped working on brave for linux but still works in private window

### ISSUE THREAD

**Issue by wratw** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#236]
### What happened?

age restricted videos no longer load in regular window but load in private window. this bug persists on latest brave release, brave beta and brave nightly. they all had same chromium version when i tested.

on this older version the script works as expected, i have downgraded for now https://github.com/brave/brave-browser/releases/tag/v1.65.89

tried on fresh mint 21.3 vm with brave having no changes except the script. tried disabling shields, using tampermonkey instead of violentmonkey, deleting all site data before opening video, changing ip, nothing worked.

google chrome with same chromium version as brave didn't have this bug. windows version of brave also worked fine. my os is ubuntu 22.04.4

### Platform

Userscript

### Version

2.5.9

### What browser are you on?

Brave

### Browser version

Version 1.64.116 Chromium: 123.0.6312.105

### Link to the video

https://www.youtube.com/watch?v=jdY89tENgmM

### About you

- [ ] I am a developer and/or have knowledge in programming

**Comment by drunkwinter** [comment]
Confirmed. This is a bug in Brave.

Filed an issue here: https://github.com/brave/brave-browser/issues/37341

### Issue:
Calling the XMLHttpRequest `send()` method after calling `open()` with this specific URL `/youtubei/v1/player` freezes the tab.

Repro:
```js
const xmlhttp = new XMLHttpRequest();
xmlhttp.open('POST', '/youtubei/v1/player', false);
xmlhttp.send();
```

**Comment by wratw** [comment]
i just checked and it also stopped working on latest windows version of brave

**Comment by drunkwinter** [comment]
Are you still able to reproduce this? After having reinstalled Brave I don't see the issue anymore.

**Comment by wratw** [comment]
yes, just tried fresh install of brave release on linux and windows, Version 1.64.116 Chromium: 123.0.6312.105
both the script and the snippet you provided still freeze the page
on private window everything works fine

**Comment by drunkwinter** [comment]
Are you willing to provide feedback on the Brave issue above? They might ask you a few things.

The griffin thing you mentioned is interesting and might be related.

**Comment by Mis1ify** [comment]
I'm also experiencing this issue. Version Brave Browser 123.1.65.108 beta Linux.

**Comment by wratw** [comment]
as mentioned here you need to disable brave-ai-chat flag https://github.com/brave/brave-browser/issues/37341#issuecomment-2048166141

since its a brave bug i think this issue can be closed

**Comment by po1st** [comment]
Happening for me as well. Found that the page freezes on "Unlock method #2 (TV Embed Player)" if that is useful.
![Screenshot 2024-05-02 202316](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/assets/89371574/75a6542a-607e-416a-a7e0-a237aef3c608)

**Comment by binnichtaktiv** [comment]
> as mentioned here you need to disable brave-ai-chat flag [brave/brave-browser#37341 (comment)](https://github.com/brave/brave-browser/issues/37341#issuecomment-2048166141)
> 
> since its a brave bug i think this issue can be closed

I can confirm that this works. Thank you but is it possible to fix this so that we dont have to disable it?

**Comment by drunkwinter** [comment]
> > as mentioned here you need to disable brave-ai-chat flag [brave/brave-browser#37341 (comment)](https://github.com/brave/brave-browser/issues/37341#issuecomment-2048166141)
> > since its a brave bug i think this issue can be closed
> 
> I can confirm that this works. Thank you but is it possible to fix this so that we dont have to disable it?

As already mentioned this is a browser bug, specifically with Brave. And an issue already has been filed on their repo.
